src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -113,7 +113,12 @@ private void assertTaskCommandBehavior(String inputCommand, String expectedMessa
         
         //Confirm the ui display elements should contain the right data
         assertEquals(expectedMessage, result.feedbackToUser);
+        
+        for(int i=0;i<model.getFilteredTaskList().size();i++){
+            System.out.print(model.getFilteredTaskList().get(i));
+        }
```

Comment:
I've changed to the OOP format, but there are many merge conflicts since I cherry picked them from the master branch. Could you help me handle the merge conflict? I'm not so confident in using the git bash terminal. I believe I'll only need the files from model manager, taskbook and markcommandtest